frontend/tests/acceptance/project-test.js

```diff
@@ -80,9 +82,6 @@ module("Acceptance | projects", function (hooks) {
 
     await click("[data-test-save]");
 
-    assert.dom("[data-test-name]").hasValue("FooBar Task 1");
-    assert.dom("[data-test-reference]").hasValue("Reference of FooBar Task 1");
-    assert.dom("[data-test-estimated-time]").hasValue("02:15");
```

Comment:
Let's test for this stuff later in the flow. Simply save the inputs and then click on the created task again.